src/table/body-cell/index.tsx

```diff
@@ -44,13 +43,14 @@ function TableCellEditable<ItemType>({
   const tdNativeAttributes = {
     'data-inline-editing-active': isEditing.toString(),
   };
+  const isFocusMoveNeededRef = useRef(false);
```

Comment:
Is this ref really necessary, or would replacing the `useEffectOnUpdate` with `useEffect` be enough?